packages/components/molecules/f-searchbox/src/components/FormStates/FormLoadingIndicator.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <div
+        v-if="shouldDisplayLoadingIndicator"
+        data-test-id="loading-spinner-indicator"
+        :class="$style['c-spinner-wrapper']">
```

Comment:
I would update this to be `c-spinner` and `c-spinner-icon` to show the hierarchy here a little better

Reply:
Fixed.